documentation/modules/operators/con-topic-operator-store.adoc

```diff
@@ -69,7 +69,7 @@ spec:
     min.insync.replicas: 2 <3>
   #...
 ----
-<1> The number of partitions for the topic. Generally, 1 partition is sufficient.
```

Comment:
Do we want to change the value in the example?

Reply:
Do we want to give any guidance on best practice for the number of partitions?

Reply:
> Do we want to give any guidance on best practice for the number of partitions?

I'm sure that would not hurt. But it is quite complex topic. So I would say it is way beyond this PR ;-)